compute_endpoint/globus_compute_endpoint/cli.py

```diff
@@ -593,13 +593,25 @@ def list_endpoints():
 @handle_auth_errors
 def delete_endpoint(*, ep_dir: pathlib.Path, force: bool, yes: bool):
     """Deletes an endpoint and its config."""
+
+    ep_conf = None
+    try:
+        ep_conf = get_config(ep_dir)
+    except Exception:
+        if not yes:
+            yes = click.confirm(
+                f"Failed to read configuration from {ep_dir}/\n"
```

Comment:
A simple catch and release as we expected, but is there any useful info in the error other than ConfigModel?  It seems like the 'no engine' is useful, should we print that (even though the error saw it previously and thus is trying to --force):
Error: 1 validation error for ConfigModel                                                                                                                                                                                                                                        
__root__                                                                                                                                                                                                                                                                         
  no engine if multi-user (type=assertion_error)

Reply:
Oooh, good call.  Now presents like:
```
$ globus-compute-endpoint delete test_mu 
(ClickException) 1 validation error for ConfigModel
__root__
  missing engine (type=assertion_error)

Failed to read configuration from /home/kevin/.globus_compute/test_mu/
  Are you sure you want to delete endpoint <test_mu>? [y/N]: 
```